app/main.py

```diff
@@ -1,6 +1,21 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
+LOCATION = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(URL, params={"q": LOCATION, "key": KEY}).json()
+
+    print(f"Current weather in Paris is: "
+          f"{response['current']['temp_c']} C, "
```

Comment:
The print statement is hardcoded to always print the weather for Paris. It would be better to use the `LOCATION` variable here, so that the print statement automatically updates if you change the location.